### ISSUE TITLE
Token has been expired or revoked

### ISSUE THREAD

**Issue by SubaKrishnan** [googleads/google-ads-php#528]
I am using Google Ads API version 6, recently I created a manager Account and linked a few Customer IDs, and then get a developer token to get access. Then I got Basic Access Approval from Google and it works. 

Then I tried to Create an OAuth Consent Screen. To complete the process I had submitted the Oauth Consent Screen for verification using the same Client ID and Client Secret. After submitting the request for verification. It shows that  "Token has been expired or revoked". Although I have the Refresh Token. But still, it's not working.

PHP Version: ^7.2.5
Laravel Version: ^7.0
Used REST Transport

![image](https://user-images.githubusercontent.com/54890552/107317258-565afa00-6ac0-11eb-8603-c305dfcb066f.png)

Can you help how to access the token back?

Thanks in advance

**Comment by fiboknacky** [comment]
Could you elaborate what you're trying to do? As you have said that you have the refresh token, I'm not sure why you're creating an OAuth consent screen here.
If you need to try doing the OAuth flow, you could try [this example](https://github.com/googleads/google-ads-php/blob/master/examples/Authentication/AuthenticateInWebApplication.php).

**Comment by SubaKrishnan** [comment]
Hi, Initially I created a manager account and got a developer token approval. 
Then I wish to create an OAuth Consent Screen. Here we don't need to link any account under the manager account. Thereby any Google account can authenticate and get their data like OAuth Playground. It will be more friendly than linking the customer account under Manager Account. That's why I am creating the OAuth consent screen.

And thanks for linking the document.

Also, I had fixed the _**Token expiring issue**_ by resetting my Client secret at the Developer Account
